src/components/noticeboard/NoticeboardAlerts.tsx

```diff
@@ -0,0 +1,11 @@
+import { Alert } from 'react-native';
+
+import { texts } from '../../config';
+
+export const NoticeboardAlerts = (alertType: string) => {
+  const buttonText = texts.noticeboard.abort,
+    message = texts.noticeboard.alerts[alertType] ?? texts.noticeboard.alerts.error,
+    title = texts.noticeboard.alerts.hint;
+
+  return Alert.alert(title, message, [{ text: buttonText, style: 'cancel' }]);
```

Comment:
the button should be "ok" when accepting some information and not "abbrechen":

|currently|will be updated to
|---|---|
|![image](https://user-images.githubusercontent.com/1942953/203593677-30b995c8-7650-4825-b839-3d69a5255292.png)|![image](https://user-images.githubusercontent.com/1942953/203593717-b34a4eb5-512a-4aee-9d3b-3212d9c66fcc.png)|